cogs/word_filter.py

```diff
@@ -65,6 +68,15 @@ def _updateFilters(self, newObj):
         finally:
             self.lock.release()
 
+    def _updateCommandBlacklist(self, newObj):

```

Comment:
Is it necessary to pass in the JSON object when all uses of it just passes `self.commandBlacklist`, which this function can also access?

Reply:
Good point, was oblivious to that.  I've made that change.